tracks/julia/exercises/run-length-encoding/mentoring.md

```diff
@@ -0,0 +1,108 @@
+## Common comments
+
+- You might not know that repeatedly concatenating strings like this causes a lot of strings to be allocated (they are not appended in place). If you want to avoid that you can use an `IOBuffer` or (less efficiently) a vector of `Char`.
+- Using `isnumeric` is safe here because the input is defined quite restrictively, but in general it is not a good idea if what you really mean is `'0' <= c <= '9'` because `isnumeric` matches a lot more characters and is a lot slower. `isdigit` is a safe alternative.
+
+
+## Example solutions
```

Comment:
Do you think we should include these even without commentary? One can look up community solutions on the side already, so I'm not sure this adds much?

Reply:
Lots of community solutions are poor, so I think it's worth providing good examples